api/scpca_portal/models/project.py

```diff
@@ -433,16 +450,28 @@ def update_project_aggregate_properties(self):
         summaries_counts = Counter()
         technologies = set()
 
+        # Get the seq_unit and technology values of bulk libraries to exclude them later
+        bulk_libraries = Library.objects.filter(modality=Modalities.BULK_RNA_SEQ)
+        bulk_seq_units = {library.metadata.get("seq_unit") for library in bulk_libraries}
+        bulk_technologies = {library.metadata.get("technology") for library in bulk_libraries}
```

Comment:
```suggestion
        # Get the seq_unit and technology values of bulk libraries to exclude them later
        bulk_libraries = Library.objects.filter(modality=Modalities.BULK_RNA_SEQ)
        bulk_seq_units = set(bulk_libraries.values_list('metadata__seq_unit', flat=True))
        bulk_technologies = set(bulk_libraries.values_list('metadata__technology', flat=True))
```

This should work as well as if these were not defined for a bulk library we would have a problem.